driver/driver.go

```diff
@@ -197,34 +197,7 @@ type Sym struct {
 	End   uint64   // virtual address of last byte in sym (Start+size-1)
 }
 
-// A UI manages user interactions.
-type UI interface {
-	// Read returns a line of text (a command) read from the user.
-	// prompt is printed before reading the command.
-	ReadLine(prompt string) (string, error)
-
-	// Print shows a message to the user.
-	// It formats the text as fmt.Print would and adds a final \n if not already present.
-	// For line-based UI, Print writes to standard error.
-	// (Standard output is reserved for report data.)
-	Print(...interface{})
-
-	// PrintErr shows an error message to the user.
-	// It formats the text as fmt.Print would and adds a final \n if not already present.
-	// For line-based UI, PrintErr writes to standard error.
-	PrintErr(...interface{})
-
-	// IsTerminal returns whether the UI is known to be tied to an
-	// interactive terminal (as opposed to being redirected to a file).
-	IsTerminal() bool
-
-	// WantBrowser indicates whether browser should be opened with the -http option.
-	WantBrowser() bool
-
-	// SetAutoComplete instructs the UI to call complete(cmd) to obtain
-	// the auto-completion of cmd, if the UI supports auto-completion at all.
-	SetAutoComplete(complete func(string) string)
-}
+type UI = plugin.UI
```

Comment:
This change seems unrelated so please remove it from this PR if it's not needed.

Reply:
Ah, looks like you say it's needed. OK, I'll look more.